sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Temporary/BasicAuthenticationPolicy.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This will be removed before first preview is released, per: https://github.com/Azure/azure-sdk-for-net/issues/19469